StreamChatClient_v3/Controllers/ChannelListController_Tests.swift

```diff
@@ -223,16 +223,27 @@ class ChannelListController_Tests: StressTestCase {
 
 private class TestEnvironment {
     var channelQueryUpdater: ChannelQueryUpdaterMock<DefaultDataTypes>?
+    var channelUpdater: ChannelUpdaterMock<DefaultDataTypes>?
     var changeAggregator: ChangeAggregator<ChannelDTO, Channel>?
     
     lazy var environment: ChannelListController.Environment = .init(channelQueryUpdaterBuilder: { [unowned self] in
-        self.channelQueryUpdater = ChannelQueryUpdaterMock(database: $0, webSocketClient: $1, apiClient: $2)
-        return self.channelQueryUpdater!
-    },
+                                                                        self
+                                                                            .channelQueryUpdater =
+                                                                            ChannelQueryUpdaterMock(database: $0,
+                                                                                                    webSocketClient: $1,
+                                                                                                    apiClient: $2)
+                                                                        return self.channelQueryUpdater!
+                                                                    },
+                                                                    channelUpdaterBuilder: { [unowned self] in
+                                                                        self.channelUpdater = ChannelUpdaterMock(database: $0,
+                                                                                                                 webSocketClient: $1,
+                                                                                                                 apiClient: $2)
+                                                                        return self.channelUpdater!
+                                                                    },
                                                                     changeAggregatorBuilder: { [unowned self] in
-        self.changeAggregator = ChangeAggregator(itemCreator: $0)
-        return self.changeAggregator!
-        })
+                                                                        self.changeAggregator = ChangeAggregator(itemCreator: $0)
+                                                                        return self.changeAggregator!
+                                                                    })
 }
```

Comment:
The formatter went really crazy here 🤦‍♂️ We need to something about it soon.

Reply:
This is actually correct behaviour. The same as in Xcode.